src/pages/login/index.tsx

```diff
@@ -45,8 +45,8 @@ const Login: FC = () => {
   useEffect(() => {
     const authenticate = async () => {
       try {
-        if (isValidSession(session)) {
-          await router.push(pages.home);
+        if (isVerifiedSession(session)) {
+          await router.push(pages.usernameSelect);
```

Comment:
So let's say I'm am just logging in with an already created account. Am I being redirected to the username select page?

Reply:
Good point

Reply:
If you are signed in with an account that does not have a username you are redirected to the username select. If you already have a username you are redirected to the home page.